### PR TITLE
Add tests, fix some bugs.

### PR DESCRIPTION
**Pull request by DanglSan**

### CODE REVIEW

File: tests/request.rs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        let contract_path = format!("{}/contract_with_lib.sol", CONTRACTS_DIR);
+        let uml_path = format!("{}/contract_with_lib.svg", UML_DIR);
+        let contract =
+            fs::read_to_string(&contract_path).expect("Error while reading contract_with_lib.sol");
+        let uml = fs::read_to_string(&uml_path).expect("Error while reading contract_with_lib.svg");
+
+        let request = SolToUmlRequest {
+            sources: BTreeMap::from([(PathBuf::from("contract_with_lib.sol"), contract)]),
+        };
+        let response = TestRequest::post()
```
Comment (leviska) [inline tests/request.rs:35]: it's okay to test via http requests, but usually people do this to test the logic of the json converting, server starting, proper http status codes in response and etc. When you just want to test the logic of your function you can just... call your function :) 
You can leave this test, it's good, but if you will add more test files, I suggest you to just call the function directly

**Comment by leviska** [comment]
looks good but I'd add some more different contracts to test, one file is kinda not enough
I suggest you add some big contract with a lot of methods to check how it is being drawn, also some contract with a lot of libraries
Contract with compile error inside it, and/or inside one of the libraries (it will fail)
Contract which imports some library, but without this library source code (it will fail)
Contract `with_a_very_long_function_name_which_can_very_sadly_ruin_some_svg_generation_or_be_cut_out_or_may_be_even_crash` :) 
Contract with function that takes a lot of arguments

File: tests/request.rs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            .set_json(&request)
+            .send_request(&app)
+            .await;
+
+        if !response.status().is_success() {
+            let status = response.status();
+            let body = read_body(response).await;
+            let message = from_utf8(&body).expect("Read body as UTF-8");
+            panic!(
+                "Invalid status code (success expected). Status: {}. Messsage: {}",
```
Comment (rimrakhimov) [inline tests/request.rs]: Nit:
```suggestion
                "Invalid status code (success expected). Status: {}. Message: {}",
```

**Review by rimrakhimov — COMMENTED** [review]
@leviska @DanglSan I suggest to add additional tests in a separate PR, as the next PR depends on that. 

The only thing that bothers me is that currently tests are not integrated into CI. Considering requirement of docker image to run integration tests, that may actually be a problem worth researching. For now, running that tests are commented from the workflow (https://github.com/blockscout/sol-to-uml/blob/main/.github/workflows/test.yml#L49) and probably we want to uncomment them. The only question is whether we want to do that in this PR or the next one. I prefer the current, but we may do as convenient

**Pull request merged**